packages/core/src/customer/customer-strategy-reducer.ts

```diff
@@ -61,6 +61,10 @@ function errorsReducer(
 
         case CustomerStrategyActionType.InitializeFailed:
             return objectMerge(errors, {
+                failedMethodIds: [
+                    ...(errors.failedMethodIds ?? []),
+                    action.meta && action.meta.methodId,
+                ],
```

Comment:
It turns out `objectMerge` does not merge two array values. Hence I merge them with `...`.